docs/ed-fi-badges/1-applying-for-an-ed-fi-badge.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Applying for an Ed-Fi Badge
+---
+
+# Applying for an Ed-Fi Badge
+
+:::tip
+
+🚧 This site is under construction. In the meantime, please see
+[Ed-Fi API Guidelines](https://edfi.atlassian.net/wiki/spaces/EFAPIGUIDE/overview)
+for more information.
```

Comment:
Not urgent - just a reminder that we'll need to update this link to a correct one, if we merge the PR without bringing the detailed content into this file.